picoCTF-web/api/apps/v1/problems.py

```diff
@@ -75,7 +75,7 @@ def get(self):
                 problem['pid'] in api.problem.get_solved_pids(tid=curr_user['tid'])
             if curr_user.get('admin', False):
```

Comment:
Wait need to revisit this why is it admin false

Reply:
iirc the intention was basically "if and only if the user is an admin, also attach all the feedback for this problem under the `reviews` key." Looking at it, `reviews` is only used within the admin problem management page to generate the % liked graphs. That page never calls `/problems/<pid>` directly currently, only `/problems?unlocked_only=false`, so it can probably be removed from this call altogether and moved within `if req['unlocked_only'] is False:` for `/problems`. It shouldn't be restricted to the users' (admin's) feedback only, though.

Reply:
A better solution is probably to cut these out entirely and add a similar admin-only ?unlocked_only to `/feedback`, and merge on pid, rather than satisfying the historical need for this `reviews` key. But idk if we want to refactor the management jsx right now

Reply:
It does get burdensome after a while, I noticed my `/problems` request was 1MB (up for 700KB) due to all the load-tested feedback. It should really just do a count instead find to stick in for admins, and yeah that'd require JSX change.

Reply:
We can leave it for now since it only affects us